source/mongorestore.txt

```diff
@@ -215,6 +215,22 @@ Starting in MongoDB 5.0, you can use ``mongorestore`` to restore
 :ref:`timeseries collections <manual-timeseries-landing>`.
 For details, see :ref:`mongorestore-example-time-series`.
 
+Using ``mongorestore`` on Atlas Clusters 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On free (``M0``) and shared (``M2`` and ``M5``) tier Atlas clusters, the
+following limitations apply:
+
+- You can't run ``mongorestore`` on the ``admin`` database. If you use
```

Comment:
I think it should also mention that it is skipped by default (if db unspecified) on these clusters